### PR TITLE
Ubuntu Clients - Structure Editing

### PR DESCRIPTION
**Pull request by Loquacity**
"backporting" changes from https://github.com/SUSE/doc-susemanager/pull/417 to 3.2

### CODE REVIEW

File: adoc/bp_chap_expanded_support.adoc [hunk 1]
```diff
@@ -430,72 +428,97 @@ Some actions are not yet supported:
 // Reason: RMT and SMT cannot mirror Debian repositories (yet) and so cannot create it in that directory the correct files.
 // We are waiting for SMT to release the feature/fix to mirror Debian repositories. When this has been done, this comment and the limitation above can be removed.
 
+== Prepare {ubuntu} Clients for Registration
```
Comment (mcalmer) [inline adoc/bp_chap_expanded_support.adoc]: Same comment as for beta, we do not prepare the  client, but the server to allow registration of these kind of clients.

File: adoc/bp_chap_expanded_support.adoc [hunk 2]
```diff
@@ -430,72 +428,97 @@ Some actions are not yet supported:
...
+For `main`:
+
+* Repository Label: ubuntu-bionic-main
+* Repository URL: http://ubuntumirror.example.com/ubuntu/dists/bionic/main/binary-amd64/
+* Repository Type: deb
 +
-----
-<user>   ALL=NOPASSWD: /usr/bin/python, /usr/bin/python2, /usr/bin/python3
-----
+For `main-updates`:
```
Comment (mcalmer) [inline adoc/bp_chap_expanded_support.adoc]: Seems the indentation is not correct.

File: adoc/bp_chap_expanded_support.adoc [hunk 3]
```diff
@@ -430,72 +428,97 @@ Some actions are not yet supported:
...
+. Synchronize the new custom channels.
+You can check the progress of your synchronization from the command line with this command:
 ----
-sudo visudo
+tail -f /var/log/rhn/reposync.log /var/log/rhn/reposync/*
 ----
-+
-Grant [command]``sudo`` access to the user by adding this line to the [filename]``sudoers`` file. Replace [systemitem]``<user>`` with the name of the user that will be used to bootsrap the client in the {webui}:
+. To use bootstrap with {ubuntu}, you will need to create a bootstrap repository.
+You can  do this from the command line with [command]``mgr-create``:
```
Comment (mcalmer) [inline adoc/bp_chap_expanded_support.adoc]: Incomplete command name. It is called 'mgr-create-bootstrap-repo'

File: adoc/bp_chap_expanded_support.adoc [hunk 4]
```diff
@@ -430,72 +428,97 @@ Some actions are not yet supported:
...
 ----
-<user>   ALL=NOPASSWD: /usr/bin/python, /usr/bin/python2, /usr/bin/python3
+mgr-create-bootstrap-repo --with-custom-channels
 ----
 
 
 
-.Procedure: Synchronizing {ubuntu} Repositories
-
+.Procedure: Granting Root User Access
```
Comment (mcalmer) [inline adoc/bp_chap_expanded_support.adoc]: I think this step needs to be done on the client. It is unclear that it needs to happen there.
All other steps above are for the server.

@mateiw is this correct?

File: adoc/bp_chap_expanded_support.adoc (hunk 2 above)
Comment (Loquacity) [reply adoc/bp_chap_expanded_support.adoc]: This is showing up correctly in the source. I think the diff  viewer is doing something strange.

Comment (mcalmer) [reply adoc/bp_chap_expanded_support.adoc]: No, please check when you click on "View File". There the indentation is wrong.

File: adoc/bp_chap_expanded_support.adoc [hunk 5]
```diff
@@ -430,72 +428,99 @@ Some actions are not yet supported:
...
-. In the [guimenu]``Create Repository`` dialog, use these values to create a new repository:
+. On the client, edit the [filename]``sudoers`` file:
 +
-* In the [guimenu]``Repository URL`` field, type the URL that contains the appropriate binary files, for example [path]``http://ubuntumirror.example.com/ubuntu/dists/bionic/main/binary-amd64/``.
-* In the [guimenu]``Repository Type`` field, select [systemitem]``deb``.
+----
+sudo visudo
+----
++
+Grant [command]``sudo`` access to the user by adding this line to the [filename]``sudoers`` file. Replace [systemitem]``<user>`` with the name of the user that will be used to bootsrap the client in the {webui}:
```
Comment (keichwa) [inline adoc/bp_chap_expanded_support.adoc]: sorry for being late...

s/bootsrap/bootstrap

Comment (Loquacity) [reply adoc/bp_chap_expanded_support.adoc]: Oops! Committed the fix directly to 3.2.

**Pull request merged**